docs/guide/http_cache.md

```diff
@@ -270,43 +205,59 @@ ezpublish:
                 purge_servers: [http://my.varnish.server:8081]
 ```
 
-!!! note "Multiple Purge Servers"
+!!! note "Invalidating Varnish cache using tokens"
 
-    If you need to set multiple purge servers, then you need to configure them in the YAML file.
+    In setups where the Varnish server IP can change (for example on platform.sh/eZ Platform Cloud),
+    you can use token-based cache invalidation via [ez_purge_acl](https://github.com/ezsystems/ezplatform-http-cache/blob/v1.0.0/docs/varnish/vcl/varnish5.vcl#L160).
```

Comment:
Why the dot inside the link?

Reply:
It is an outside link, so on the page it will look like there is a space between the link and the full stop. This is a little workaround to prevent that.